### PR TITLE
[DE] Update `Client/Menu tips`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Menu_tips/de.md [hunk 1]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
+- Die meisten der Tastaturkürzel im Spiel sind konfigurierbar und auf die eigenen Bedürfnisse anpassbar. Schlage die Tastenbelegungen in den Eingabeeinstellungen nach!
+- Wenn das HUD im Spiel verborgen ist, kannst du `Strg` halten, um es vorübergehend zu sehen!
+- Das HUD im Spiel kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
+- Ziehe Bilder in den Skin-Editor, um sie schnell zu laden!
+- Durch Mod-Voreinstellungen gelangst du einfacher zu deinen bevorzugten Mod-Kombinationen!
+- Viele Mods haben Konfigurationseinstellungen, die ihre Funktionsweise drastisch ändern. Klicke in der Modauswahl auf den Button `Mod Customisation`, um die Einstellungen zu sehen!
+- Drücke `Strg` + `Shift` + `R`, um einen zufälligen Skin auszuwählen!
+- Drücke `Strg` + `Shift` + `F` für die FPS-Anzeige. Aber lass dich nicht davon ablenken!
+- Drücke während des Ansehens eines Replays `Strg` + `H` für die Replay-Einstellungen!
+- Klicke mit der rechten Maustaste auf einen Score in der Rangliste, um dessen Mods zu kopieren!
```
Comment (The-Last-Cookie) [inline wiki/Client/Menu_tips/de.md:121]: kopieren:
- anzuwenden
- zu übertragen

🤔

File: wiki/Client/Menu_tips/de.md [hunk 2]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
 - Die meisten Webinhalte (Profile, Ranglisten etc.) sind über die Symbole in der Symbolleiste direkt im Spiel verfügbar!
 - Erhalte mehr Details, verstecke oder lösche eine Beatmap, indem du mit der rechten Maustaste auf ihr Dialogfeld in der Songauswahl klickst!
 - Alle Löschoperationen sind bis zum Verlassen des Spiels nur vorübergehend. Stelle versehentlich gelöschte Inhalte in den Wartungseinstellungen wieder her!
 - Probiere das "Playlists"-System aus, das Nutzer ihre eigenen, bleibenden Bestenlisten kreieren lässt!
 - Mit `Strg` + `F11` kannst du Statistiken über Frames und Threads erhalten!
 - Werfe einen Blick unter die Haube der Leistungsindikatoren und aktiviere die ausführliche Leistungsprotokollierung mit `Strg` + `F2`!
+- Während eines Replays kannst du mit `Leertaste` pausieren!
+- Die meisten der Tastaturkürzel im Spiel sind konfigurierbar und auf die eigenen Bedürfnisse anpassbar. Schlage die Tastenbelegungen in den Eingabeeinstellungen nach!
+- Wenn das HUD im Spiel verborgen ist, kannst du `Strg` halten, um es vorübergehend zu sehen!
+- Das HUD im Spiel kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
```
Comment (The-Last-Cookie) [inline wiki/Client/Menu_tips/de.md:114]:
```suggestion
- Das Spieldesign kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
```
🤔

File: wiki/Client/Menu_tips/de.md [hunk 3]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
 ## osu!(lazer)
 
-Diese Tipps erscheinen beim Start von [osu!(lazer)](/wiki/Client/Release_stream/Lazer), vor der Intro-Sequenz. Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
+Diese Tipps erscheinen unten im Hauptmenü von [osu!(lazer)](/wiki/Client/Release_stream/Lazer). Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
 
-- Du kannst überall `Strg` + `T` drücken, um die Toolbar zu aktivieren oder zu deaktivieren!
+- Du kannst überall `Strg` + `T` drücken, um die Symbolleiste zu aktivieren oder zu deaktivieren!
 - Du kannst überall `Strg` + `O` drücken, um auf die Optionen zuzugreifen!
-- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, das Spiel zu pausieren und den Skin zu ändern!
+- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin während des Spiels zu ändern!
```
Comment (FreezyLemon) [inline wiki/Client/Menu_tips/de.md:97]: The EN version changes from "while playing" to "while watching autoplay". I think we should reflect that here

File: wiki/Client/Menu_tips/de.md [hunk 4]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
 - Mit jedem Update kommen neue Features hinzu. Bleibe immer auf dem Laufenden!
 - Wenn dir die Bedienelemente zu groß oder zu klein sind, versuche, die UI-Skalierung in den Einstellungen anzupassen!
 - Versuche, die `Bildschirmskalierung` anzupassen, um den Bereich der Benutzeroberfläche oder des Gameplays zu ändern, sogar im Vollbildmodus!
-- Was früher "[osu!direct](/wiki/osu!supporter#osu!direct)" war, ist nun wie auf der Webseite für alle Nutzer verfügbar. Du kannst es überall mit `Strg` + `D` aufrufen!
-- In Replays kann durch das Ziehen des Cursors auf der Schwierigkeitsleiste am unteren Rand des Bildschirms gesprungen werden!
+- Was früher "[osu!direct](/wiki/osu!supporter#osu!direct)" war, ist nun wie auf der Webseite für alle Nutzer verfügbar. Du kannst es überall mit `Strg` + `B` aufrufen!
+- In Replays kann durch das Ziehen des Cursors oder durch die linke und rechte Pfeiltaste auf der Fortschrittsleiste am unteren Rand des Bildschirms gesprungen werden!
```
Comment (FreezyLemon) [inline wiki/Client/Menu_tips/de.md:102]: The sentence sounds weird after the addition regarding arrow keys. How about

```suggestion
- Du kannst durch Replays springen, indem du die linke oder rechte Pfeiltaste drückst oder den Cursor über die Fortschrittsleiste unten ziehst!
```

File: wiki/Client/Menu_tips/de.md [hunk 5]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
-- Probiere, in der Modauswahl herunterzuscrollen, um einen Haufen neuer spaßiger Mods zu entdecken!
+- Probiere, in der Modauswahl nach rechts zu scrollen, um einen Haufen neuer spaßiger Mods zu entdecken!
 - Die meisten Webinhalte (Profile, Ranglisten etc.) sind über die Symbole in der Symbolleiste direkt im Spiel verfügbar!
 - Erhalte mehr Details, verstecke oder lösche eine Beatmap, indem du mit der rechten Maustaste auf ihr Dialogfeld in der Songauswahl klickst!
 - Alle Löschoperationen sind bis zum Verlassen des Spiels nur vorübergehend. Stelle versehentlich gelöschte Inhalte in den Wartungseinstellungen wieder her!
 - Probiere das "Playlists"-System aus, das Nutzer ihre eigenen, bleibenden Bestenlisten kreieren lässt!
 - Mit `Strg` + `F11` kannst du Statistiken über Frames und Threads erhalten!
 - Werfe einen Blick unter die Haube der Leistungsindikatoren und aktiviere die ausführliche Leistungsprotokollierung mit `Strg` + `F2`!
+- Während eines Replays kannst du mit `Leertaste` pausieren!
+- Die meisten der Tastaturkürzel im Spiel sind konfigurierbar und auf die eigenen Bedürfnisse anpassbar. Schlage die Tastenbelegungen in den Eingabeeinstellungen nach!
```
Comment (FreezyLemon) [inline wiki/Client/Menu_tips/de.md:112]: I don't think I have ever heard `Tastaturkürzel` before.. `Tastenkürzel` seems to be the standard in the wiki I think? `Tastenkombination` is also sometimes used. Don't have a strong stance on this, but I did notice it was unusual

File: wiki/Client/Menu_tips/de.md (hunk 2 above)
Comment (FreezyLemon) [reply wiki/Client/Menu_tips/de.md:114]: Both are not very descriptive.. Is using `Gameplay` a problem? I think it's a common enough word, and `Gameplay-HUD` would be obvious to most I assume.

File: wiki/Client/Menu_tips/de.md [hunk 6]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
+- Wenn das HUD im Spiel verborgen ist, kannst du `Strg` halten, um es vorübergehend zu sehen!
+- Das HUD im Spiel kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
+- Ziehe Bilder in den Skin-Editor, um sie schnell zu laden!
+- Durch Mod-Voreinstellungen gelangst du einfacher zu deinen bevorzugten Mod-Kombinationen!
+- Viele Mods haben Konfigurationseinstellungen, die ihre Funktionsweise drastisch ändern. Klicke in der Modauswahl auf den Button `Mod Customisation`, um die Einstellungen zu sehen!
+- Drücke `Strg` + `Shift` + `R`, um einen zufälligen Skin auszuwählen!
+- Drücke `Strg` + `Shift` + `F` für die FPS-Anzeige. Aber lass dich nicht davon ablenken!
+- Drücke während des Ansehens eines Replays `Strg` + `H` für die Replay-Einstellungen!
```
Comment (FreezyLemon) [inline wiki/Client/Menu_tips/de.md:120]:
```suggestion
- Drücke beim Ansehen eines Replays `Strg` + `H` für die Replay-Einstellungen!
```

File: wiki/Client/Menu_tips/de.md (hunk 1 above)
Comment (FreezyLemon) [reply wiki/Client/Menu_tips/de.md:121]: How about

```suggestion
- Klicke mit der rechten Maustaste auf einen Score in der Rangliste, um dieselben Mods zu aktivieren!
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]

File: wiki/Client/Menu_tips/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/Client/Menu_tips/de.md:97]: Even though it is obscured and not directly mentioned, this line refers to the `Autoplay` mod. Taking some freedom in translation here is not problematic in my opinion.

My other suggestion would be

```suggestion
- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin im automatischen Spielmodus zu ändern!
```

or even more directly

```suggestion
- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin zu ändern während die Mod `Autoplay` aktiv ist!
```

File: wiki/Client/Menu_tips/de.md (hunk 4 above)
Comment (The-Last-Cookie) [reply wiki/Client/Menu_tips/de.md:102]: One `unten` too many.

File: wiki/Client/Menu_tips/de.md (hunk 5 above)
Comment (The-Last-Cookie) [reply wiki/Client/Menu_tips/de.md:112]:
```suggestion
- Die meisten der Tastenkürzel im Spiel sind konfigurierbar und auf die eigenen Bedürfnisse anpassbar. Schlage die Tastenbelegungen in den Eingabeeinstellungen nach!
```

File: wiki/Client/Menu_tips/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Client/Menu_tips/de.md:114]:
```suggestion
- Das Gameplay-HUD kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
```

File: wiki/Client/Menu_tips/de.md (hunk 3 above)
Comment (FreezyLemon) [reply wiki/Client/Menu_tips/de.md:97]: > Even though it is obscured and not directly mentioned, this line refers to the Autoplay mod. Taking some freedom in translation here is not problematic in my opinion.

I'm not against this in general, but in this specific case the English version was changed to explicitly mention the autoplay mod. So I think also mentioning it in the translation is preferable.

Of your two suggestions, the second one feels better IMHO

File: wiki/Client/Menu_tips/de.md (hunk 4 above)
Comment (FreezyLemon) [reply wiki/Client/Menu_tips/de.md:102]: Sure, that was just a suggestion. Feel free to do keep the original or do something else if it sounds better to you.

File: wiki/Client/Menu_tips/de.md (hunk 5 above)
Comment (FreezyLemon) [reply wiki/Client/Menu_tips/de.md:112]: I'd prefer it

File: wiki/Client/Menu_tips/de.md (hunk 4 above)
Comment (The-Last-Cookie) [reply wiki/Client/Menu_tips/de.md:102]: On rereading, I get what this `unten` does there, sorry. I like your suggestion more but the `unten` still seems weird to me. Now I would either leave it out entirely or add this part:

```suggestion
- Du kannst durch Replays springen, indem du die linke oder rechte Pfeiltaste drückst oder den Cursor über die Fortschrittsleiste am unteren Bildschirmrand ziehst!
```

File: wiki/Client/Menu_tips/de.md [hunk 7]
```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
...
 ## osu!(lazer)
 
-Diese Tipps erscheinen beim Start von [osu!(lazer)](/wiki/Client/Release_stream/Lazer), vor der Intro-Sequenz. Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
+Diese Tipps erscheinen unten im Hauptmenü von [osu!(lazer)](/wiki/Client/Release_stream/Lazer). Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
 
-- Du kannst überall `Strg` + `T` drücken, um die Toolbar zu aktivieren oder zu deaktivieren!
+- Du kannst überall `Strg` + `T` drücken, um die Symbolleiste zu aktivieren oder zu deaktivieren!
 - Du kannst überall `Strg` + `O` drücken, um auf die Optionen zuzugreifen!
-- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, das Spiel zu pausieren und den Skin zu ändern!
+- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin zu ändern während die Mod `Autoplay` aktiv ist!
```
Comment (The-Last-Cookie) [inline wiki/Client/Menu_tips/de.md:97]: While we are at it, we might as well link to the corresponding article

```suggestion
- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin zu ändern während die Mod [Autoplay](/wiki/Gameplay/Game_modifier/Auto) aktiv ist!
```

**Review by The-Last-Cookie — COMMENTED** [review]
What I added in the previous review round, for visibility:

> On rereading, I get what this `unten` does there, sorry. I like your suggestion more but the `unten` still seems weird to me. Now I would add this part:
>
> Du kannst durch Replays springen, indem du die linke oder rechte Pfeiltaste drückst oder den Cursor über die Fortschrittsleiste am unteren Bildschirmrand ziehst!

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**